internal/hcsoci/resources_lcow.go

```diff
@@ -6,18 +6,28 @@ package hcsoci
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"os"
 	"path"
 	"path/filepath"
 	"strings"
 
 	"github.com/Microsoft/hcsshim/internal/log"
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
 )
 
 const lcowMountPathPrefix = "/mounts/m%d"
+const lcowNvidiaMountPath = "/nvidia"
+
+func getNvidiaGpuVhdPath(coi *createOptionsInternal) (string, error) {
+	path := filepath.Join(filepath.Dir(os.Args[0]), "gpu.vhd")
```

Comment:
Huh? I thought we talked about this being a config value many times :). What did I miss about the final decision?

Reply:
Yep! I'm working on that! Just wanted the rest of the code to get reviewed as p0 while I'm working on demo related things. I can try to get that in today if you'd like?